cmd/serve/serve.go

```diff
@@ -194,6 +213,8 @@ func NewCommand(
 				srvErrs <- server.Run(ctx)
 			}()
 
+			go consumer.Consume()
```

Comment:
this needs improving either way...possibly leverage existing srvErrs? or just check error period

Reply:
Updated this a little bit, ive created a new channel of errors within the consumer similar to how eventing is setup so that if its a consumer error we can explicitly show it is in 'reason'. Also updated this to use a go func for the purpose of at least ignoring the error since errors are being handled elsewhere